site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -2,15 +2,15 @@
 layout: 'layouts/doc-post.njk'
 title: About extension service workers
 description: Extension service workers are an extension's central event handler. That makes them different from web service workers. 
-date: 2023-05-03
+date: 2023-06-03
```

Comment:
```suggestion
date: 2023-05-03
```

Reply:
There's two reasons I changed this back. First, this isn't how we indicate content updates. Second, we don't do this for typo fixes.